### DISCUSSION TITLE
posts/Go/http/

### DISCUSSION THREAD

**Discussion by Q1mi** [Q1mi/BlogComments#283] [Announcements]
https://www.liwenzhou.com/posts/Go/http/ 

 李文周的Blog Go语言 socket 网络编程 TCP UDP net http go

**Comment by flashcatt** [comment]
老师看不懂啊，怎么学习啊呜呜呜呜

**Reply by seabra2** [reply]
这是对于网络请求的一种封装，记住用法就行，要是想深入研究可以去看net/http的底层代码
